File: C1.UWP.FlexChart/VB/DataManipulation/LibCommon/Boolean2StackingConverter.vb
Imports C1.Chart

Public Class Boolean2StackingConverter
    Implements IValueConverter

    Private Function IValueConverter_Convert(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.Convert
        Dim b As Boolean = CType(value, Boolean)
        If b Then
            Return Stacking.Stacked
        End If
        Return Stacking.None
    End Function

    Private Function IValueConverter_ConvertBack(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.ConvertBack
        Dim s As Stacking = CType(value, Stacking)
        If s = Stacking.Stacked Then
            Return True
        End If
        Return False
    End Function
End Class

File: C1.UWP.FlexChart/CS/DataManipulation/LibCommon/Boolean2StackingConverter.cs
using C1.Chart;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Windows.UI.Xaml.Data;

namespace DataManipulation
{
    public class Boolean2StackingConverter : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, string language)
        {
            bool b = System.Convert.ToBoolean(value);
            if (b)
            {
                return Stacking.Stacked;
            }
            return Stacking.None;
        }

        public object ConvertBack(object value, Type targetType, object parameter, string language)
        {
            Stacking s = (Stacking)value;
            if (s == Stacking.Stacked)
            {
                return true;
            }
            return false;
        }
    }
}
